packages/salesforcedx-vscode-core/src/commands/util/parameterGatherers.ts

```diff
@@ -0,0 +1,19 @@
+import {
```

Comment:
I like this approach to break the different command utility methods into files based on the interfaces they implement. Are we going to do this for the rest of the classes that are also implementing these interfaces. These changes now leave us with 2 main places where these type of classes are hosted, under the `commands/util` folder and on the `commands.ts` file.

Reply:
Correct, I will be filing work to finish breaking up `commands.ts` into respective files.